common/pkg/libartifact/store/reference.go

```diff
@@ -0,0 +1,100 @@
+//go:build !remote
+
+package store
+
+import (
+	"context"
+
+	"go.podman.io/image/v5/docker/reference"
+)
+
+type ArtifactReference struct {
+	reference.Named
```

Comment:
Does this intentionally expose the full interface of `reference.Named`?

Reply:
ok done.  privatized it and exposed `Name()` and `String()` methods.